modules/099-ceph-csi/crds/cephcsi.yaml

```diff
@@ -0,0 +1,131 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: cephcsi.deckhouse.io
+  labels:
+    heritage: deckhouse
+    module: ceph-csi
+    app: ceph-csi
+spec:
+  group: deckhouse.io
+  scope: Cluster
+  names:
+    plural: cephcsi
+    kind: CephCSI
+  preserveUnknownFields: false
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      schema: &schema
+        openAPIV3Schema:
+          type: object
+          description: |
+            Ceph cluster connection parameters and StorageClasses configuration
+          required:
+            - spec
+          properties:
+            spec:
+              type: object
+              required:
+                - clusterID
+                - userID
+                - userKey
+                - monitors
+              properties:
+                clusterID:
+                  description: |
+                    Ceph FSID/UUID
+                  type: string
+                userID:
+                  description: |
+                    Username without `client.`
+                  type: string
+                userKey:
+                  description: |
+                    Ceph auth key corresponding to userID
+                  type: string
+                monitors:
+                  description: |
+                    List of ceph-mon IP addresses in the format `10.0.0.10:6789`
+                  type: array
+                  items:
+                    type: string
+                rbd:
+                  type: object
+                  properties:
+                    storageClasses:
+                      description: |
+                        Description of StorageClasses for Rados Block Device (RBD)
+                      type: array
+                      items:
+                        type: object
```

Comment:
Are there any required fields?

Reply:
Yes, there should be required field. Added required section.